### PR TITLE
Part 1 completed : main screen design

### PR DESCRIPTION
**Pull request by ceddddy**
# What we did in this phase:

- Added linters and dependencies 

- Added a header and navigation menu

- Added the tittle heading, helper text  

- Added social media links

### CODE REVIEW

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @ceddddy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive READme file, kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a guide.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <h2 class="navItem">Welcome</h2>
```
Comment (aimalamiri) [inline index.html:17]: You did a good job 👌but here is how to improve it.
1. Please make sure to put this in the `a` tag, so the user can always navigate to the home page.
2. It's required to follow the exact Figma template, so please make sure to adapt your design with the original one. As you can see the navbar in the original design has no background image, and also please try to update your sizing and spacing as well. These kinds of simple things can significantly improve the design.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          <p> I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+          <ul class="socilaLinks">
+            <li><a href="#" target="_blank"><img src="/images/twitter.png" /></a></li>
+            <li><a href="#" target="_blank"><img src="/images/Linkedin icon.png"/></a></li>
+            <li><a href="#" target="_blank"><img src="/images/medium.png" /></a></li>
+            <li><a href="#" target="_blank"><img src="/images/github.png" /></a></li>
+            <li><a href="#" target="_blank"><img src="/images/angellist.png"/></a></li>
+          </ul>
+
```
Comment (aimalamiri) [inline index.html:36]: You did a great job! 👌but as I mentioned it's very important to follow the Figma design as much as possible, especially the font, sizing, spacing, and colors. So please make sure to update the font type and size in here.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @ceddddy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The header section is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Lizdev-05,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**